superset/assets/utils/common.js

```diff
@@ -39,8 +39,19 @@ export function getParamFromQuery(query, param) {
   return null;
 }
 
-export function getLink(baseUrl, params) {
-  return baseUrl + '?' + params.join('&');
+export function getQueryLink(query, callback) {
```

Comment:
I actually think that keeping query as an object in params makes a cleaner interface and make this func more reusable in other places